### DISCUSSION TITLE
FaceSwapLab ported to ComfyUi or use of safesensors converter.

### DISCUSSION THREAD

**Discussion by KINGLIFER** [glucauze/sd-webui-faceswaplab#173] [Ideas]
This is 10 times better than the competition but auto1111 is to introductory for me now. I am on Comfy and made awesome workflows but miss dearly this software. How can we port this to a node or at the very least make a converter for the ton of models I made with this?

Thank you everyone.

**Comment by dougvan1701** [comment]
+1
All other "face swap" utilities pale in comparison to Faceswaplab. The ability to build face models from multiple images sets this apart. Would love to use it outside of the beginner Auto1111 framework.
